settings.gradle

```diff
@@ -22,5 +22,6 @@ include('lang',
         'pts',
         'extensions',
         'partiql-grammar',
-        'lib:partiql-isl'
+        'lib:partiql-isl',
+        'partiql-tests-runner'
```

Comment:
I suggest we create top-level `test` directory which holds the various testing modules. 

```
└── test
    ├── partiql-pts
    ├── partiql-test-runner
    └── partiql-testscript
```

Reply:
The `partiql-tests` github submodule would be included in `test/` rather than with the partiql-test-runner

Reply:
Sure, added a `test` directory which holds the test modules. Also renamed the `pts` and `testscript` dirs to include the `partiql` prefix.

And moved `partiql-tests` github submodule to under `tests`